signal/signal.go

```diff
@@ -19,6 +22,50 @@ var (
 	started int32
 )
 
+// systemdNotifyReady notifies systemd about LND being ready, logs the result of
+// the operation or possible error. Besides logging, systemd being unavailable
+// is ignored.
+func systemdNotifyReady() error {
+	notified, err := daemon.SdNotify(false, daemon.SdNotifyReady)
+	if err != nil {
+		err := fmt.Errorf("failed to notify systemd %v (if you aren't"+
```

Comment:
nit: add spaces before line break, otherwise this will read `if you aren'trunning systemd clear the environment variableNOTIFY_SOCKET`.

Reply:
Nice catch!